viewing_party/main.py

```diff
@@ -0,0 +1,178 @@
+from statistics import mode
+
+# Wave 1
+
+def create_movie(title, genre, rating):
+    """
+    Creates dictionary of movie title, genre, and rating
+    """
+    movie_dict = {}
+    if title and genre and rating:
+        movie_dict["title"] = title
+        movie_dict["genre"] = genre
+        movie_dict["rating"] = rating
+        return movie_dict
+    else:
+        return None
```

Comment:
You can clean this up a little with a dictionary literal:

```suggestion
    if title and genre and rating:
        return {
            "title": title,
            "genre": genre,
            "rating": rating
        }
    else:
        return None
```